packages/camera/camera/ios/Classes/FLTThreadSafeTextureRegistry.m

```diff
@@ -6,7 +6,7 @@
 #import "QueueUtils.h"
 
 @interface FLTThreadSafeTextureRegistry ()
-@property(nonatomic, strong) NSObject<FlutterTextureRegistry> *registry;
+@property(nonatomic, weak) NSObject<FlutterTextureRegistry> *registry;
```

Comment:
If we want to release the texture registry we should release the whole wrapper object.

Reply:
sorry for the delay.
didn't get you here (fairly a beginner at ObjC)